iot/api-client/manager/src/test/java/com/example/cloud/iot/examples/ManagerIT.java

```diff
@@ -52,10 +53,16 @@
   private static final String CLOUD_REGION = "us-central1";
   private static final String ES_PATH = "resources/ec_public.pem";
   private static final String PROJECT_ID = System.getenv("GOOGLE_CLOUD_PROJECT");
-  private static final String REGISTRY_ID = "java-reg-" + (System.currentTimeMillis() / 100L);
+  private static final String REGISTRY_ID =
+      "java-reg-"
+          + (System.currentTimeMillis() / 100L);
```

Comment:
If two or more of these started at the same time, there could be duplicates.  Would that be ok?  If not, you should also add a UUID here.

Reply:
there is clearRegistries function that deletes the registries based on created Date (from oldest to new ones).

Reply:
You might be better off using `java-reg-UUID-TimeMillis`.